packages/browser-service/src/browser.service.ts

```diff
@@ -52,16 +71,19 @@ export class BrowserService {
     return this.walletPage;
   }
 
-  getBrowserContext() {
-    return this.browserContextService.browserContext;
+  getNetworkConfig() {
```

Comment:
I believe we can set `this.networkConfig = this.options.netoworkConfig` since there is the way with default private options - 'constructor(private options: BrowserServiceOptions) {}'